build/cmake/lib/webview/CMakeLists.txt

```
@@ -34,6 +34,21 @@ endif()
 
 if(APPLE)
     wx_lib_link_libraries(webview PUBLIC "-framework WebKit")
+elseif(WXMSW)
+    if(wxUSE_WEBVIEW_EDGE)
+        wx_lib_include_directories(webview PRIVATE ${PROJECT_SOURCE_DIR}/3rdparty/webview2/build/native/include)
```

In case the directory contains spaces.

```suggestion
        wx_lib_include_directories(webview PRIVATE "${PROJECT_SOURCE_DIR}/3rdparty/webview2/build/native/include")
```